src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/GenotypeAlleleCounts.java

```diff
@@ -661,7 +661,9 @@ GenotypeAlleleCounts copy() {
 
 
     public void forEachAlleleIndexAndCount(final IntBiConsumer action) {
-        new IndexRange(0, distinctAlleleCount).forEach(n -> action.accept(sortedAlleleCounts[2*n], sortedAlleleCounts[2*n+1]));
+        for (int n = 0; n < distinctAlleleCount; n++) {
```

Comment:
Did this show up in the profiler? I'm generally for optimizing away Java Stream calls wherever it makes a difference.

Reply:
This change was a shot in the dark from before I realized the cache-cleverness of the original code, but it doesn't show up in the profiler.  I'm sure it would if the code was an `IntStream`, but `IndexRange` is just a simple object with two integers -- there's nothing lurking behind it's constructor there way there is for Java streams.